_config/pom.yml

```diff
@@ -18,23 +18,23 @@ latestFinal:
     droolsExpert_documentationPdf: http://docs.jboss.org/drools/release/6.4.0.Final/drools-docs/pdf/drools-expert-docs.pdf
 
 latest:
-    version: 7.0.0.Beta1
-    releaseDate: 2016-07-05
+    version: 7.0.0.Beta2
+    releaseDate: 2016-10-03
 
-    jbpmBinZip: http://download.jboss.org/jbpm/release/7.0.0.Beta1/jbpm-7.0.0.Beta1-bin.zip
-    jbpmExamplesZip: http://download.jboss.org/jbpm/release/7.0.0.Beta1/jbpm-7.0.0.Beta1-examples.zip
-    jbpmInstallerZip: http://download.jboss.org/jbpm/release/7.0.0.Beta1/jbpm-7.0.0.Beta1-installer.zip
-    updatesite: http://download.jboss.org/jbpm/release/7.0.0.Beta1/updatesite/
+    jbpmBinZip: http://download.jboss.org/jbpm/release/7.0.0.Beta2/jbpm-7.0.0.Beta2-bin.zip
+    jbpmExamplesZip: http://download.jboss.org/jbpm/release/7.0.0.Beta2/jbpm-7.0.0.Beta2-examples.zip
+    jbpmInstallerZip: http://download.jboss.org/jbpm/release/7.0.0.Beta2/jbpm-7.0.0.Beta2-installer.zip
+    updatesite: http://download.jboss.org/jbpm/release/7.0.0.Beta2/updatesite/
 
-    releaseNotesVersion: http://docs.jboss.org/jbpm/release/7.0.0.Beta1/jbpm-docs/html/ch25.html
-
-    UserGuide: http://docs.jboss.org/jbpm/release/7.0.0.Beta1/jbpm-docs/html/
-    JavaDocs: http://docs.jboss.org/drools/release/7.0.0.Beta1/kie-api-javadoc/index.html
+    UserGuide: http://docs.jboss.org/jbpm/release/7.0.0.Beta2/jbpm-docs/html_single
+    JavaDocs: http://docs.jboss.org/drools/release/7.0.0.Beta2/kie-api-javadoc/index.html
     Nightly: https://jenkins-kieci.rhcloud.com/job/kie-docs/lastStableBuild/artifact/jbpm-docs/target/docbook/publish/en-US/html/index.html
 
-    droolsExpert_documentationHtmlSingle: http://docs.jboss.org/drools/release/7.0.0.Beta1/drools-docs/html_single/index.html
-    droolsExpert_documentationHtml: http://docs.jboss.org/drools/release/7.0.0.Beta1/drools-docs/html/index.html
-    droolsExpert_documentationPdf: http://docs.jboss.org/drools/release/7.0.0.Beta1/drools-docs/pdf/drools-expert-docs.pdf
+    releaseNotesVersion: http://docs.jboss.org/jbpm/release/7.0.0.Beta2/jbpm-docs/html_single/#_jbpmreleasenotes
+
+    droolsExpert_documentationHtmlSingle: http://docs.jboss.org/drools/release/7.0.0.Beta2/drools-docs/html_single/index.html
+    droolsExpert_documentationHtml: http://docs.jboss.org/drools/release/7.0.0.Beta2/drools-docs/html/index.html
+    droolsExpert_documentationPdf: http://docs.jboss.org/drools/release/7.0.0.Beta2/drools-docs/pdf/drools-expert-docs.pdf
```

Comment:
Due to certain issues with asciidoc-maven-plugin, we currently have only the html_single docs. The other two links should be temporarily removed until we fix that.